### PR TITLE
New Package: dtrx-7.1

### LINKED ISSUE #1335: Package Request: dtrx
"Do The Right Extraction."

http://brettcsmith.org/2007/dtrx/

This is a tiny Python script used to automate the extraction of various archive and compression formats. It's a minor convenience package.

### PR DESCRIPTION
**Pull request by logenkain**
This closes #1335

### CODE REVIEW

File: srcpkgs/dtrx/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'dtrx'
+pkgname=dtrx
+version=7.1
+revision=1
+noarch=yes
+build_style=python-module
+pycompile_module="dtrx"
+makedepends="python"
```
Comment (Gottox) [inline srcpkgs/dtrx/template]: python is a `hostmakedepends`.

**Comment by xtraeme** [comment]
python -> python-setuptools
makedepends -> hostmakedepends

**Comment by logenkain** [comment]
I was looking at the Deluge Template and noticed that python-setuptools is in all three depends lines.  Is that how I should have this template?

**Comment by xtraeme** [comment]
Commonly `python-setuptools` in `hostmakedepends` is enough for pure python modules (no C extensions).

**Comment by xtraeme** [comment]
I noticed some things from the output of travis builds:

- `pycompile_module` is unnecessary, remove it.
- the ARM packages are empty, i.e no files in DESTDIR.

**Pull request merged**